src/main/java/org/terasology/minimap/rendering/nui/layers/MinimapGrid.java

```diff
@@ -94,11 +96,8 @@ public TextureRegion load(Block block) {
 
                     BlockPart blockPart = BlockPart.TOP;
 
-                    // TODO: security issues
-                    // WorldAtlas worldAtlas = CoreRegistry.get(WorldAtlas.class);
-                    // float tileSize = worldAtlas.getRelativeTileSize();
-
-                    float tileSize = 16f / 256f; // 256f could be replaced by textureAtlas.getWidth();
```

Comment:
probably something to do with CoreRegistry. there are some long term plans to replace it with gestalt-DI.